suite-native/module-check-backup/src/components/CheckBackupTutorialStep2.tsx

```diff
@@ -27,15 +30,27 @@ type NavigationProps = StackNavigationProps<
     DeviceCheckBackupStackRoutes.CheckBackupTutorial
 >;
 
+const checkBackupUnsupportedDeviceModels: Array<DeviceModelInternal | null> = [
+    DeviceModelInternal.T1B1,
+];
+
 export const CheckBackupTutorialStep2 = ({
     currentStepIndex,
 }: WalletBackupTutorialNumberedStepProps) => {
     const navigation = useNavigation<NavigationProps>();
+    const deviceModel = useSelector(selectDeviceModel);
 
     const { applyStyle } = useNativeStyles();
     const { showToast } = useToast();
 
     const navigateToCheckBackup = () => {
+        if (deviceModel && checkBackupUnsupportedDeviceModels.includes(deviceModel)) {
```

Comment:
waiting for @shenkys response if this is correct entry point https://www.figma.com/design/yY9Zl2jXLKXVV3CwuBmaMU?node-id=2595-4323&m=dev#1326660712